contributors.json

```diff
@@ -670,5 +670,9 @@
     "country": "India",
     "name": "Aarushi"
   }
+  "Code-hunter-star":{
```

Comment:
Can you order this in ascending order alphabetically according to the GitHub user name and fix the JSON syntax?

Reply:
yes sure